apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.html

```diff
@@ -32,7 +32,7 @@ <h4 class="modal-title bg pull-left">Adjust Payment</h4>
                         <div class="col-sm-3">
                             <div class="form-group">
                                 <label for="name">Adjustment Voucher<span class="text-danger">*</span></label>
-                                <div [class.error-box]="entry.uniqueName === '' && adjustAdvancePaymentForm?.form.controls['voucherName'+entryIdx]?.touched">
+                                <div class="select-error" [class.error-box]="entry.uniqueName === '' && adjustAdvancePaymentForm?.form.controls['voucherName'+entryIdx]?.touched">
```

Comment:
Class `select-error` removes the border by setting it width to 0px, in error scenario `error-box` class is added which adds a red border around the div. Please check if `error-box` class is still working properly after your change

Reply:
I discussed with Rohit about this error red color,  Rohit will work on in it

Reply:
Sadique I have added 
[class.error-box]="entry.uniqueName === '' && adjustAdvancePaymentForm?.form.controls['voucherName'+entryIdx]?.touched"
That was working 
after you have added class="select-error" please check is it working?